README.rst

```diff
@@ -34,6 +34,14 @@ In the Kinto project settings
     kinto.attachment.folder = {bucket_id}/{collection_id}
     kinto.attachment.keep_old_files = true
 
+
+If you want uploaded files to get gzipped when stored:
+
+.. code-block:: ini
+
+    kinto.attachment.gzipped = true
```

Comment:
Does it means that all the collection having attachment will be gzipped? I wish we could have a per resource matching for this setting. We are doing it for the signer credentials for instance: https://github.com/Kinto/kinto-signer#multiple-certificates